Reneiski_Homework1/ReneiskiSelenium11/Tests/BaseTest.cs

```diff
@@ -0,0 +1,28 @@
+﻿using NUnit.Framework;
+using ReneiskiSelenium11.Common.Drivers;
+using ReneiskiSelenium11.Data;
+using ReneiskiSelenium11.PageObjects;
+
+namespace ReneiskiSelenium11.Tests
+{
+    public class BaseTest
+    {
+        public BaseTest()
+        {
+        }
+
+        [OneTimeSetUp]
+        public void OneTimeSetup()
+        {
+            WebDriverFactory.InitializeDriver();
+            WebDriverFactory.Driver.Navigate().GoToUrl(TestSettings.PageUrl);
+            GenericPages.BasePage.GoToElemntsPage();
```

Comment:
what if you will need to write tests where you would need to go to Widgets page from StartPage?

Reply:
Fixed